boards/opentitan/src/main.rs

```diff
@@ -319,9 +319,9 @@ unsafe fn setup() -> (
     // Need to enable all interrupts for Tock Kernel
     chip.enable_plic_interrupts();
     // enable interrupts globally
-    csr::CSR
-        .mie
-        .modify(csr::mie::mie::msoft::SET + csr::mie::mie::mtimer::SET + csr::mie::mie::mext::SET);
+    csr::CSR.mie.modify(
+        csr::mie::mie::msoft::SET + csr::mie::mie::mtimer::CLEAR + csr::mie::mie::mext::SET,
```

Comment:
Is mtimer enabled by default? If not, it seems less confusing to just leave it out entirely, or at least add a note that we do not enable that interrupt because we instead get that signal via the PLIC

Reply:
I think we should explicitly clear this. It's possible that the bootROM enabled it, plus this makes it obvious for anyone reading the code that we disable the timer interrupt